knimin/tests/test_pm_shotgun_pool_handlers.py

```diff
@@ -0,0 +1,148 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The LabAdmin Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from unittest import main
+from functools import partial
+from random import choice
+from traceback import format_exc
+
+import numpy as np
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+from knimin import db
+
+
+class TestPMShotgunPool(TestHandlerBase):
+    def setUp(self):
```

Comment:
You can remove this function and the next one - those are already added on the "TestHandlerBase" that should also fix the errors that I sent you before